app/controllers/miq_alert_controller.rb

```diff
@@ -252,7 +252,8 @@ def show
       @ems = ExtManagementSystem.find_by(:id => @alert.expression[:options][:ems_id])
     end
     if @alert.expression.kind_of?(Hash) && @alert.expression[:eval_method]
-      MiqAlert.expression_options(@alert.expression[:eval_method]).each do |eo|
+      @expression_options = MiqAlert.expression_options(@alert.expression[:eval_method])
```

Comment:
why make this an instance variable?  is it used elsewhere?

Reply:
The same class method was called again in its 'show' file. Hence, used an instance variable to reuse it.